style.css

```diff
@@ -0,0 +1,66 @@
+*{
+   /* background-image: url(./survey2.jpg); */
+   image-resolution: 50%;
+   
+}
+#main{
+    background-image: url(./survey2.jpg);
```

Comment:
Please consider using the following attributes such that the background image is positioned correctly
```
  background-image: url(./survey2.jpg);
  background-size: Cover;
  background-repeat: no-repeat;
```
This will help you position the image correctly in the background.